csharp/ql/test/library-tests/aliases/aliases2.expected

```diff
@@ -1,3 +1,9 @@
 | Program.cs:18:21:18:22 | c1 | Assembly1.dll:0:0:0:0 | Class |
+| Program.cs:18:21:18:22 | c1 | Assembly2.dll:0:0:0:0 | Class |
```

Comment:
@hvitved I'm not sure whether this is a desired change in behaviour. The reason for the change is that the local variable has a type given by a `typeref` which is no longer qualified with the assembly id. Therefore, the type of the local variable maps to all matching types, which is three different types in this case.

To recover the original behaviour, the extractor simply needs to populate a `type` as opposed to a `typeref` in the local variable's `type` field.

Reply:
I don't think types of local variables should be treated specially, what do you think?